### PR TITLE
Az3190 deploy pipeline

### PR DESCRIPTION
**Pull request by uday-ssc**

### CODE REVIEW

**Comment by lucstjean-ssc** [comment]
All my comments can be found in the original PR: https://github.com/ssc-spc-ccoe-cei/gcp-documentation/pull/4

File: Landing Zone Operations/Building_Using_Automation.md [hunk 1]
```diff
@@ -0,0 +1,198 @@
...
+    **You will need to push to main when running git push**
+    
+    `git push --set-upstream origin main`
+
+1. Excellent ! You have your `tier1-infra` repository ready for next steps.
+
+
+## 2. Pull the tools submodule
+
+Populate the `tools` folder with the content from the `gcp-tools` repository. Ensure you are still at the root of the `tier1-infra` repo and execute `modupdate.sh`
```
Comment (lucstjean-ssc) [inline Landing Zone Operations/Building_Using_Automation.md:113]: The code block is displaying correctly. I think you're missing the `1.` at the start of the line.

Comment (uday-ssc) [reply Landing Zone Operations/Building_Using_Automation.md:113]: Fixed

File: Landing Zone Operations/Building_Using_Automation.md [hunk 2]
```diff
@@ -0,0 +1,198 @@
...
+2. **Important** Customize the new file with the approriate values for the landing zone you are building.
+
+## 4. Running the setup-kcc automated script
+
+The script creates a project, the FW settings, a Cloud router, a Cloud NAT, a private service connect endpoint and the Anthos Config Controller cluster. It also creates a root-sync.yaml file.
+    ```
+    bash setup-kcc.sh <PATH TO .ENV FILE>
+    ```
+
+Once the script has completed, you need to move the `root-sync.yaml` file into the `bootstrap/<env>` folder of the `tier1-infra` repo.
```
Comment (lucstjean-ssc) [inline Landing Zone Operations/Building_Using_Automation.md:136]: This code block is also not displaying correctly when viewing the file/rich diff. If you are not using a `1.` then I don't think you need a tab space for your code block.

Comment (lucstjean-ssc) [reply Landing Zone Operations/Building_Using_Automation.md:136]: If your not using a `1.` then I don't think you need a tab space for your code block.

Comment (uday-ssc) [reply Landing Zone Operations/Building_Using_Automation.md:136]: changed it to use 1. , i think its better to follow the same convention throughout

**Review by lucstjean-ssc — APPROVED** [review]
Excellent. /lgtm

File: Landing Zone Operations/Building_Using_Automation.md [hunk 3]
```diff
@@ -0,0 +1,198 @@
...
+
+1. Generate hydrated files
+
+    Refer to the `Generate hydrated files` section of the [Changing.md](Changing.md#Generate%20hydrated%20files)
+
+1. Add changes to repository
+    
+    Refer to the `Add changes to repository` section of the [Changing.md](Changing.md#Add%20changes%20to%20repository).
+    
+    **You will need to push to main when running git push**
```
Comment (alaincormier-ssc) [inline Landing Zone Operations/Building_Using_Automation.md:187]: should be done with a PR, can be refined in future ticket

**Review by alaincormier-ssc — APPROVED** [review]

**Comment by ovidiuasiminei-ssc** [comment]
lgtm

**Pull request merged**